### PR TITLE
add GDC performance tests

### PR DESCRIPTION
**Pull request by tomkralidis**
Add forgotten test

### CODE REVIEW

File: tests/global_discovery_catalogue/test_gdc_performance.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+        time.sleep(5 * sleep_factor)
+
+    time.sleep(10 * sleep_factor)
+
+    end = time.time()
+    elapsed = (end - start) - (10 * sleep_factor)
+
+    run_api_tests()
+
+    assert elapsed < 300
```
Comment (jmaxmarno) [inline tests/global_discovery_catalogue/test_gdc_performance.py:72]: i think this doesn't account for the sleep that is in the for w2p loop. at the very least it does not account for the sleep_factor that is applied there. 5 minutes should be plenty but you might want to also add the sleep factor here? 300*sleep_factor?

File: tests/global_discovery_catalogue/test_gdc_performance.py [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+                link.get('type') == 'application/geo+json']):
+            query_url = link['href']
+            break
+
+    assert query_url is not None
+
+    LOGGER.info('Querying items endpoint')
+    r = requests.get(query_url).json()
+
+    assert r['numberMatched'] == 6
```
Comment (jmaxmarno) [inline tests/global_discovery_catalogue/test_gdc_performance.py:96]: where does the value 6 come from? does that need to change dynamically too?

Comment (tomkralidis) [reply tests/global_discovery_catalogue/test_gdc_performance.py:96]: 6 is the number of valid WCMP2 records that are in the test data (that should be published).

Comment (jmaxmarno) [reply tests/global_discovery_catalogue/test_gdc_performance.py:96]: copy - no further questions

File: tests/global_discovery_catalogue/test_gdc_performance.py (hunk 1 above)
Comment (tomkralidis) [reply tests/global_discovery_catalogue/test_gdc_performance.py]: Updated

**Review by jmaxmarno — APPROVED** [review]

**Pull request merged**